CONTRIBUTING.md

```diff
@@ -38,17 +54,78 @@ OS            | Supported tests
 OS X          | Android, iOS
 Windows       | Android
 
+### Test descriptions
+
+The tests first build the app.
+
+They then check if the required emulators are currently running. If running only Android tests, it checks for a running Android emulator, and if running only iOS tests, it checks for a running iOS simulator. If running both, it will check for both.
```

Comment:
The first sentence seems sufficient. Probably can remove from `If running only Android tests ... it will check for both.`